schema/bom-1.5.proto

```diff
@@ -209,30 +209,34 @@ enum ExternalReferenceType {
   EXTERNAL_REFERENCE_TYPE_ATTESTATION = 16;
   // An enumeration of identified weaknesses, threats, and countermeasures, dataflow diagram (DFD), attack tree, and other supporting documentation in human-readable or machine-readable format
   EXTERNAL_REFERENCE_TYPE_THREAT_MODEL = 17;
+  // The defined assumptions, goals, and capabilities of an adversary.
+  EXTERNAL_REFERENCE_TYPE_ADVERSARY_MODEL = 18;
```

Comment:
Changing the values in an existing enum? bad idea! 
This breaks backwards compatibility for implementers and consumers in so many ways. 

from an implementor's perspective: 
No i would have to mantain a ENUM for CDX<1.5 and another one for CDX==1.5 - and they are in no way compatible. and type-save programming in java would not even be possible.

Reply:
> Changing the values in an existing enum? bad idea!

Agreed. However, I have inserted them into a list of enums that are specific to v1.5, so the reordering does not break anything.

Reply:
for example: enum value "EXTERNAL_REFERENCE_TYPE_DISTRIBUTION_INTAKE" changed drom a `18` to a `20`.

this is unacceptable from an implementor's perspective.
Now i would have to maintain one ENUM for CDX<1.5 and another one for CDX==1.5 - and they are in no way compatible. and type-save programming in java would not even be possible.


please add new names with new values, do not change existing key-value-combinations.

Reply:
@jkowalleck,  inspecting `bom-1.4.proto` I see that External Reference enum values only go up to 14.  ie,  15 and higher is what is **new** to schema 1.5.  Thus, there is no incompatibility when tweaking 18 to 20.... right?